testdata/good_config.json

```diff
@@ -1,7 +1,8 @@
 {
     "signing": {
         "default": {
-            "expiry": "168h"
+            "expiry": "168h",
+            "name_whitelist": "^.*cloudflare.com"
```

Comment:
For consistency, this should probably match the previous note about the regex, as should the line below.